pennylane/vqe/vqe.py

```diff
@@ -288,12 +288,33 @@ def __matmul__(self, H):
         if isinstance(H, Hamiltonian):
             coeffs2 = H.coeffs
             terms2 = H.ops
-
```

Comment:
Worthwhile updating the changlog (it can be separate from the other qaoa changes, e.g., a new bullet point in "improvements")

Reply:
addressed here: 5a7e94452c5e76ebffe376a482dcff1ed7b1804c